### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by Jonathan0827** [community/community#30248] [GitHub Education]
I'm currently in an elementary school in South Korea, but our school doesn't have a school email account. Do I need to get an account through the Department of Education?

**Comment by melvincwng** [comment]
@Jonathan0827, Yes to qualify for the GitHub Student Developer Pack, you need to have a school-registered email account.

However, please refer to this link from [GitHub Documentation](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student) on the full requirements needed to apply for the student developer pack:

> - Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution
> - Have a verifiable school-issued email address or upload documents that prove your current student status
> - Have a [GitHub personal account](https://docs.github.com/en/articles/signing-up-for-a-new-github-account)
> - Be at least 13 years old

You should also satisfy those terms (as quoted above) before applying for the student pack. As you indicated you're in elementary school, I don't think that qualifies you for the student developer pack.

Hope this clarifies and answers your question 😄!
